config.sample.yaml

```diff
@@ -83,3 +82,7 @@ developer_settings:
 # Used by the API and Resource processor application to change log level
 # Can be "ERROR", "WARNING", "INFO", "DEBUG"
 #  logging_level: "INFO"
+
+# If you want to use TRE_URL to point to your local TRE API instance or be configured to another cloud provider
+# uncomment and set this variable
+#  tre_url: __CHANGE_ME__
```

Comment:
Hmm, so this used to be used to register bundles before the TRE CLI existed.

I think it is still used in the e2e tests - see `e2e_tests/config.py`, to point to the TRE API you want to run them against.

Reply:
Ah, sorry, github is hiding part of the file. I see now, thats fine.